README.rst

```diff
@@ -35,3 +35,39 @@ strategies more easily available.  There are currently two strategies available:
     insert ' ' '  '
     equal 'This is some text.' 'This is some text.'
     delete '  ' ''
+
+
+Tokenization
+============
+
+By default Deltas performs tokenization by regexp text splitting. 
+We included CJK tokenization functionality. If text consists of at least 1/4
+Japanse or Korean symbols it is Tokenized by language specific Tokenizer.
+Else, Chinese Tokenizer is used.
+
+Chinese Tokenizer - Jieba
+Japanese Tokenizer - Sudachi
+Korean Tokenizer - KoNLPy(Okt)
+
+:CJK Tokenization example:
+import mwapi
+import deltas
+import deltas.tokenizers
+
+session = mwapi.Session("https://zh.wikipedia.org")
+doc = session.get(action="query", prop="revisions",
+                  titles="中国", rvprop="content", rvslots="main",
+                  formatversion=2)
+text = doc['query']['pages'][0]['revisions'][0]['slots']['main']['content']
+
+tokenized_text = deltas.tokenizers.wikitext_split.tokenize(text)
+tokenized_text_cjk = deltas.tokenizers.wikitext_split_w_cjk.tokenize(text)
+
+FOR IMPROVED JAPANESE TOKENIZER ACCURACY INSTALL FULL DICTIONARY
+================================================================
+
+```
+pip install sudachidict_full
+# and link sudachi to dict
+sudachipy link -t full
+```
```

Comment:
When converting to Markdown, make sure to add this line to setup.py: https://github.com/wikimedia/revscoring/blob/master/setup.py#L42

Reply:
markdown instruction converted and line added